test/egeoip_tests.erl

```diff
@@ -59,17 +62,27 @@ egeoip() ->
     {ok, IpAddressLong} = egeoip:ip2long("207.145.216.106"),
     {ok, IpAddressLong} = egeoip:ip2long(<<207,145,216,106>>),
     {ok, R} = egeoip:lookup(IpAddressLong),
+    Region
+        = case application:get_env(egeoip, dbfile) of
+              {ok, city} -> <<"CA">>;
+              _ -> <<>>
+          end,
     #geoip{country_code = "US",
            country_code3 = "USA",
            country_name = "United States",
-           region = <<"CA">>,
+           region = Region,
            _ = _} = R,
```

Comment:
You mean change the already-existing tests to not do pattern-matching? And instead use something like
`?assertMatch(#geoip{country_code = "US",
           country_code3 = "USA",
           country_name = "United States",
           region = Region1,
           _ = _}, R1)`